graphs/possible_bipartition.py

```diff
@@ -5,8 +5,33 @@ def possible_bipartition(dislikes):
     """ Will return True or False if the given graph
         can be bipartitioned without neighboring nodes put
         into the same partition.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(V + E)
+        Space Complexity: O(n)
```

Comment:
✨ Correct for a breadth first search implementation, but I would suggest you define your variables particularly because you use different variables to represent the same thing here. V and n both represent the number of vertices/nodes and E represents the number of edges.